cdap-ui/app/cdap/components/PlusButtonModal/PlusButtonModal.scss

```diff
@@ -43,6 +46,40 @@
           height: 100%;
           width: 100%;
         }
+
+        /*
+          width of each card = 100% - (padding of parent) - (combined margin of all cards in the row)
+          margin for each card is 5px (as mentioned above).
+          So the combined margin for all cards in say ,
+            4 column layout : 5px + (5px + 5px) + (5px + 5px) + 5px = (3 * 10px)
+            5 column layout : 5px + (5px + 5px) + (5px + 5px) + (5px + 5px) + 5px = (4 * 10px)
+            6 column layout : 5px + (5px + 5px) + (5px + 5px) + (5px + 5px) + (5px + 5px) + 5px = (5 * 10px)
+            n column layout : (n-1 * 10px);
+        */
+
+        @media (min-width: 1601px) {
+          .resourcecenter-entity-card {
+            width: calc((100% - 20px - 10px - (5 * 10px)) / 4);
+          }
+        }
+
+        @media (min-width: 1201px) and (max-width: 1600px) {
+          .resourcecenter-entity-card {
+            width: calc((100% - 20px - 10px - (4 * 10px)) / 3);
+          }
+        }
+
+        @media (min-width: 992px) and (max-width: 1200px) {
```

Comment:
nit: Maybe change this `min-width` to `993px` now that the `max-width` of the one-entity layout is `992px`? Right now they're both `992px`